src/components/menus/SnippetList.jsx

```diff
@@ -16,11 +16,13 @@ const makeMenuItems = _.memoize((userSnippets, onClick) => {
 });
 
 const SnippetList = ({ titles, onClick }) => {
+  const menuItems = makeMenuItems(titles, onClick);
   return (
     <MenuItem
       primaryText="My Snippets"
       rightIcon={<ArrowDropRight />}
-      menuItems={makeMenuItems(titles, onClick)}
+      disabled={menuItems ? false : true}
```

Comment:
I also find it more readable this way. I'm willing to change it though, if you can explain that what you propose is objectively better.